static/js/play.js

```diff
@@ -114,13 +141,26 @@ let app = new Vue({
             this.hidePreview();
             // Game logic for sprint mode:
 
-            if (this.path.length == 0 || this.path[this.path.length - 1] != page) {
-                this.path.push(page);
+            let loadTimeSeconds = loadTime / 1000;
+
+            if (this.path.length == 0 || this.path[this.path.length - 1]["article"] != page) {
+                // Update path
+                let timeElapsed = (Date.now() - this.startTime) / 1000;
+                this.path.push({
+                    "article": page,
+                    "timeReached": timeElapsed,
+                    "loadTime": loadTimeSeconds
+                });
+
+                // Update partial run information for non-lobby runs
+                if (this.lobbyId === null) {
```

Comment:
Actually my point about potentially submitting multiple runs isn't right, but we should also update for lobby runs I think